_includes/pbc/all/install-features/202403.0/install-glue-api/install-the-checkout-glue-api.md

```diff
@@ -0,0 +1,946 @@
+
+
+
```

Comment:
do we need these spaces?

Reply:
This was moved from the previous version of this IG. @andriitserkovnyi, can we delete these empty lines?

Reply:
will do